examples/gameroom/daemon/Dockerfile-installed-focal

```diff
@@ -98,11 +98,10 @@ RUN apt-get update \
  && apt-get -f -y install
 
 # Fetch the XO smart contract
-RUN curl -OLsS https://build.sawtooth.me/job/Sawtooth-Hyperledger/job/sawtooth-sdk-rust/job/master/lastSuccessfulBuild/artifact/build/scar/*zip*/scar.zip
+RUN curl -OLsS http://files.splinter.dev/scar/xo_0.5.1.scar
```

Comment:
Do we not have https for this?

Reply:
Oops. I forgot that this is fronted with Cloudfront for for SSL.